src/init/starship.fish

```diff
@@ -37,4 +37,4 @@ builtin functions -e fish_mode_prompt
 set -gx STARSHIP_SHELL "fish"
 
 # Set up the session key that will be used to store logs
-set -gx STARSHIP_SESSION_KEY (random 10000000000000 9999999999999999)
+set -gx STARSHIP_SESSION_KEY (string sub -s1 -l16 (random)(random)(random)(random)(random)0000000000000000)
```

Comment:
Any chance we can add a comment explaining the reason we don't use `random [min] [max]`?

Reply:
I added a comment explaining the reason.